src/SFA.DAS.AssessorService.Web.Staff/Validators/Roatp/AddOrganisationValidationErrorMessages.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SFA.DAS.AssessorService.Web.Staff.Validators.Roatp
+{
+    public static class AddOrganisationValidationErrorMessages
+    {
+        public static string InvalidProviderTypeId = "Tell us what the new training provider is";
```

Comment:
needs a tech debt story to get these words in a resx maybe

Reply:
Can do this when we play the rest of the validation messages